iroha-cli/client.cpp

```diff
@@ -37,12 +38,31 @@ namespace iroha_cli {
     auto pb_tx = factory.serialize(model_tx);
     // Send to iroha:
     iroha::protocol::ToriiResponse response;
-    auto stat = client_.Torii(pb_tx, response);
+    auto stat = command_client_.Torii(pb_tx, response);
 
     return response.validation() ==
                    iroha::protocol::STATELESS_VALIDATION_SUCCESS
                ? OK
                : NOT_VALID;
   }
 
+  iroha::protocol::QueryResponse CliClient::sendQuery(std::string json_query) {
+    iroha::model::converters::JsonQueryFactory serializer;
+
+    auto query_opt = serializer.deserialize(std::move(json_query));
+
+    iroha::protocol::QueryResponse query_response;
+
+    if (not query_opt.has_value()) {
+      iroha::protocol::ErrorResponse er;
+      er.set_reason(iroha::protocol::ErrorResponse::WRONG_FORMAT);
+      query_response.mutable_error_response()->CopyFrom(er);
+      return query_response;
+    }
+
+    query_client_.Find(query_opt.value(), query_response);
```

Comment:
Same as above - save return value and check `.error_code`, `.error_details`, or `.error_message` to display gRPC error.